spec/support/command_helpers.rb

```diff
@@ -157,6 +157,9 @@ def create_app_if_not_exists(app, deploy: false, image_before_deploy_count: 0, i
   end
 
   def run_cpflow_command(*args, raise_errors: false) # rubocop:disable Metrics/MethodLength
+    program_name_before = $PROGRAM_NAME
+    $PROGRAM_NAME = package_name
```

Comment:
What's the reason for this? And where does `$PROGRAM_NAME` come from? Thor?

Reply:
It comes from Thor - https://github.com/rails/thor/blob/a43d92fad7ebd77d359b7b96eb3db8a73ef9057c/lib/thor/base.rb#L772

Without this tweak we get this output (`rspec` i/o `cpflow`):
```
cpflow commands:
  rspec apply-template TEMPLATE [TEMPLATE] ... [TEMPLATE] -a, --app=APP_NAME              # Applies application-specific configs from templates
  rspec build-image -a, --app=APP_NAME                                                    # Builds and pushes the image to Control Plane
  rspec cleanup-images -a, --app=APP_NAME                                                 # Deletes all images for an app that either exceed the max quanti...
  rspec cleanup-stale-apps -a, --app=APP_NAME                                             # Deletes the whole app (GVC with all workloads, all volumesets a...
  rspec config                                                                            # Displays config for each app or a specific app
  rspec copy-image-from-upstream -a, --app=APP_NAME -t, --upstream-token=UPSTREAM_TOKEN   # Copies an image (by default the latest) from a source org to th...
  rspec delete -a, --app=APP_NAME                                                         # Deletes the whole app (GVC with all workloads, all volumesets a...
  rspec deploy-image -a, --app=APP_NAME                                                   # Deploys the latest image to app workloads, and runs a release s...
  rspec doctor                                                                            # Runs validations
  rspec env -a, --app=APP_NAME                                                            # Displays app-specific environment variables
  rspec exists -a, --app=APP_NAME                                                         # Shell-checks if an application (GVC) exists, useful in scripts
  rspec generate                                                                          # Creates base Control Plane config and template files
  rspec help [COMMAND]                                                                    # Describe available commands or one specific command
  rspec info                                                                              # Displays the diff between defined/available apps/workloads (app...
  rspec latest-image -a, --app=APP_NAME                                                   # Displays the latest image name
  rspec logs -a, --app=APP_NAME                                                           # Light wrapper to display tailed raw logs for app/workload syntax
  rspec maintenance -a, --app=APP_NAME                                                    # Checks if maintenance mode is on or off for an app
  rspec maintenance:off -a, --app=APP_NAME                                                # Disables maintenance mode for an app
  rspec maintenance:on -a, --app=APP_NAME                                                 # Enables maintenance mode for an app
  rspec maintenance:set-page PAGE_URL -a, --app=APP_NAME                                  # Sets the page for maintenance mode
  rspec open -a, --app=APP_NAME                                                           # Opens the app endpoint URL in the default browser
  rspec open-console -a, --app=APP_NAME                                                   # Opens the app console on Control Plane in the default browser
  rspec promote-app-from-upstream -a, --app=APP_NAME -t, --upstream-token=UPSTREAM_TOKEN  # Copies the latest image from upstream, runs a release script (o...
  rspec ps -a, --app=APP_NAME                                                             # Shows running replicas in app
  rspec ps:restart -a, --app=APP_NAME                                                     # Forces redeploy of workloads in app
  rspec ps:start -a, --app=APP_NAME                                                       # Starts workloads in app
  rspec ps:stop -a, --app=APP_NAME                                                        # Stops workloads in app
  rspec ps:wait -a, --app=APP_NAME                                                        # Waits for workloads in app to be ready after re-deployment
  rspec run COMMAND -a, --app=APP_NAME                                                    # Runs one-off interactive or non-interactive replicas (analog of...
  rspec setup-app -a, --app=APP_NAME                                                      # Creates an app and all its workloads
  rspec terraform                                                                         # Terraform commands
  rspec version                                                                           # Displays the current version of the CLI

Options:
  -o, [--org=ORG_NAME]             # Organization name
  -d, [--verbose], [--no-verbose]  # Shows detailed logs
      [--trace], [--no-trace]      # Shows trace of API calls. WARNING: may contain sensitive data
```

Reply:
Reverted this change